test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -211,7 +211,8 @@ void ChainBranch::restoreFromHistory(size_t _importBlockNumber)
 void ChainBranch::forceBlockchain(string const& chainname)
 {
 	s_tempBlockchainNetwork = dev::test::TestBlockChain::s_sealEngineNetwork;
-	dev::test::TestBlockChain::s_sealEngineNetwork = stringToNetId(chainname);
+	if (chainname != "default")
+		dev::test::TestBlockChain::s_sealEngineNetwork = stringToNetId(chainname);
```

Comment:
What kind of problem does this solve?   Is it possible to get this commented?

Reply:
Just restoring to original logic. perhaps this could be optimized yes.

Reply:
I see.  Interesting.